scripts/beck/st/error_injection.py

```diff
@@ -164,128 +176,156 @@ def ReadMetaData(offset):
 	with open(filename, "rb") as mdata:
 		mdata.seek(offset)
 		data=binascii.hexlify((mdata.read(8))[::-1])
-		print("** Metadata at offset {} is {}".format(hex(offset), data))
+		logger.info("** Metadata at offset {} is {}".format(hex(offset), data))
 
 def induceCorruption(recordType, noOfCorr):
-	print(recordType)
-	print("No of Error want to induce: {}".format(noOfCorr))
+	count = 0
+	read_metadata_file()
+	logger.info(recordType)
+	logger.info("No of Error want to induce: {}".format(noOfCorr))
 	lookupList = recordDict[recordType]
 	if (len(lookupList) & noOfCorr) == 0:
```

Comment:
This is bit AND operation.
I don't understand this condition.

Reply:
Yes if the length is zero and the number of errors user input is zero then only fulfill this condition